docs/DeveloperGuide.md

```diff
@@ -511,6 +512,56 @@ Step 3. The user executes `delete_archived A0123456A` command to delete student
 Given below is the sequence diagram for `delete_archived` command:
 ![DeleteArchiveSequenceDiagram](images/DeleteArchiveSequenceDiagram.png)
 
+--------------------------------------------------------------------------------------------------------------------
+### Set Weakness Threshold Feature
+
+This is a new command to designate students as being "weak" or not based on their grades. `thresholdGrade` is a value 
+within `Grade`. 
+By default, we have set C+ as the `thresholdGrade`, meaning that a student with grade lower than or equal to C+ is
+displayed with a weak marker next to their name (as shown below).
+
+<img src="images/weak_label.png" alt="Weak Label" width="200">
+
+
+The command `set weak g/GRADE` followed by the grade parameter allows the instructor to set a different grade as the
```

Comment:
Might be good to state that this is not applicable for archived students

Reply:
Done in latest commit, and also in UG